### PR TITLE
chore: bump rc-rate version

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | chore: bump rc-rate version Star Component Refactor to FC |
| 🇨🇳 Chinese | 使用 FC 重构 class component |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
changlog 应包含 整个minor的改动说吗

**Comment by li-jia-nan** [comment]
> changlog 应包含 整个minor的改动说明

update～

**Comment by yoyo837** [comment]
应该在 `feature` 更新吧?

File: package.json [hunk 1]
```diff
@@ -137,7 +137,7 @@
     "rc-pagination": "~3.2.0",
     "rc-picker": "~3.2.4",
     "rc-progress": "~3.4.1",
-    "rc-rate": "~2.9.0",
+    "rc-rate": "~2.10.0",
```
Comment (zombieJ) [inline package.json:140]: 啧啧，CI 过了。看起来没问题

**Review by zombieJ — APPROVED** [review]

**Pull request merged**